projects/storefrontstyles/scss/_components.scss

```diff
@@ -27,6 +28,7 @@ $selectors: mergeAll(
     $content-components-whitelist,
     $product-components-whitelist,
     $cart-components-whitelist,
+    $organization-components-whitelist,
```

Comment:
MyOrg styles were moved to my org lib. Changes in this file should be dropped I guess